cmd/credit.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"time"
+
+	"github.com/aidanaden/canvas-sync/internal/app/credit"
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+)
+
+// represents the credit command
+var creditCmd = &cobra.Command{
+	Use:   "credits",
+	Short: "Show a list of contributors to canvas-sync project.",
+	Long: `Show a list of contributors to canvas-sync project. First one is list of user's with at least one commit in 
+the past week. Then a list of all time contributors sorted by number of commits descending.
+
+Examples:
+  canvas-sync credits`,
+	Run: func(cmd *cobra.Command, args []string) {
+        println()
+
+        // Past week contributors
+        pterm.FgGreen.Println("Past week contributors:")
+        sevenDaysAgo := time.Now().AddDate(0, 0, -7)
+        latestContributors := credit.ContributorsSince(sevenDaysAgo)
+        for _, c := range latestContributors {
+            pterm.FgGreen.Printfln("- %s", c)
+        }
+        println()
+
+        // Top 500 contributors
+        pterm.FgCyan.Println("All contributors:")
+        allContributors := credit.Contributors()
+        for _, c := range allContributors {
+            pterm.FgCyan.Printfln("- %s", c)
+        }
+        println()
+	},
```

Comment:
is it possible to print the following instead?
- new contributors (first pr/commit within the last week)
- active contributors (top 10 contributors within the last month)

i think this'll incentivise both active development and new contributors to join

Reply:
I changed the code to calculate and print "active contributors". But the "new contributors" you said is not simple. Do you have any idea, how we should do it correctly and efficiently? There are no APIs that can do it. We can calculate it manually by receiving all the commits, but it is not efficient and can be impossible in the near future due to the increasing number of commits.

Reply:
i think using some of the the apis from https://docs.github.com/en/rest/metrics/statistics?apiVersion=2022-11-28 could be useful